web/vtadmin/src/api/http.ts

```diff
@@ -42,9 +43,13 @@ export const vtfetch = async (endpoint: string, options: RequestInit = {}): Prom
         }
 
         const { REACT_APP_VTADMIN_API_ADDRESS } = process.env;
-        const url = `${REACT_APP_VTADMIN_API_ADDRESS}${endpoint}`;
         const opts = { ...vtfetchOpts(), ...options };
 
+        // To support variable API endpoints, check if an api address is set as a cookie before
```

Comment:
These changes are critical for @planetscale/vtadmin package, but I can see it being added to the OS VTAdmin later.  This file probably won't change too much in the future as the main structure of HTTP requests won't change too much for VTAdmin OS.